exchanges/bittrex.js

```diff
@@ -351,34 +351,312 @@ Trader.prototype.getTrades = function(since, callback, descending) {
   this.bittrexApi.getmarkethistory({ market: params.currencyPair }, process);
 }
 
-Trader.getCapabilities = function () {
+Trader.getCapabilities = function() {
   return {
     name: 'bittrex',
     slug: 'bittrex',
-    currencies: ['BTC', 'ETH', 'USDT'],
+    currencies: ['BITCNY', 'BTC', 'ETH', 'USDT'],
     assets: [
-      'BTC', 'BCC','ETH','NEO','BCC','PAY'
+      '1ST', '2GIVE', '8BIT', 'ABY', 'ADC', 'ADT', 'ADX', 'AEON', 'AGRS', 'AM',
+      'AMP', 'AMS', 'ANS', 'ANT', 'APEX', 'APX', 'ARB', 'ARCH', 'ARDR', 'ARK',
+      'AUR', 'BAT', 'BAY', 'BCY', 'BITB', 'BITCNY', 'BITS', 'BITZ', 'BLC',
+      'BLITZ', 'BLK', 'BLOCK', 'BNT', 'BOB', 'BRK', 'BRX', 'BSD', 'BSTY', 'BTA',
+      'BTC', 'BTCD', 'BTS', 'BURST', 'BYC', 'CANN', 'CCN', 'CFI', 'CLAM',
+      'CLOAK', 'CLUB', 'COVAL', 'CPC', 'CRB', 'CRBIT', 'CRW', 'CRYPT', 'CURE',
+      'CVC', 'DAR', 'DASH', 'DCR', 'DCT', 'DGB', 'DGC', 'DGD', 'DMD', 'DOGE',
+      'DOPE', 'DRACO', 'DTB', 'DTC', 'DYN', 'EBST', 'EDG', 'EFL', 'EGC', 'EMC',
+      'EMC2', 'ENRG', 'ERC', 'ETC', 'ETH', 'EXCL', 'EXP', 'FAIR', 'FC2', 'FCT',
+      'FLDC', 'FLO', 'FRK', 'FSC2', 'FTC', 'FUN', 'GAM', 'GAME', 'GBG', 'GBYTE',
+      'GCR', 'GEMZ', 'GEO', 'GHC', 'GLD', 'GNO', 'GNT', 'GOLOS', 'GP', 'GRC',
+      'GRS', 'GRT', 'GUP', 'HKG', 'HMQ', 'HYPER', 'HZ', 'INCNT', 'INFX', 'IOC',
+      'ION', 'IOP', 'J', 'JWL', 'KMD', 'KORE', 'KR', 'LBC', 'LGD', 'LMC', 'LSK',
+      'LTC', 'LUN', 'LXC', 'MAID', 'MAX', 'MCO', 'MEC', 'MEME', 'METAL', 'MLN',
+      'MND', 'MONA', 'MTL', 'MTR', 'MUE', 'MUSIC', 'MYR', 'MYST', 'MZC', 'NAUT',
+      'NAV', 'NBT', 'NEOS', 'NET', 'NEU', 'NLG', 'NMR', 'NTRN', 'NXC', 'NXS',
```

Comment:
NEOS should be NEO. Was just trying out BTC/NEO market watcher on Bittrex. So glad you made this PR, as I was about to do it myself!